src/edu/washington/escience/myria/util/SamplingType.java

```diff
@@ -0,0 +1,9 @@
+package edu.washington.escience.myria.util;
+
+/**
+ * Enumeration of supported sampling types.
+ */
+public enum SamplingType {
+  // WithReplacement, WithoutReplacement
+  WR, WoR
```

Comment:
Why not just the long forms of the names? It would be more to type but would be self-documenting...